coopertunes/utils.py

```diff
@@ -196,6 +204,18 @@ def calc_n_params(module):
     return sum(p.numel() for p in module.parameters())
 
 
+def save_sample(file_path, sampling_rate, audio):
+    """Helper function to save sample
+
+    Args:
+        file_path (str or pathlib.Path): save file path
+        sampling_rate (int): sampling rate of audio (usually 22050)
+        audio (torch.FloatTensor): torch array containing audio in [-1, 1]
+    """
+    audio = (audio.numpy() * 32768).astype("int16")
```

Comment:
` * 32768?`

Reply:
Not exactly sure what it was too, but now it is named macro ;)